README.md

```diff
@@ -15,7 +15,7 @@ If you want to use OpenLaw core in your Scala project, here is how to add it to
 resolvers += "https://openlaw.bintray.com/openlaw-core"
 
 //add the dependency
-libraryDependencies += "org.openlaw" % "openlaw-core" & "<last version>"
+libraryDependencies += "org.openlaw" % "openlaw-core_2.12" & "<last version>"
```

Comment:
my bad, instead of that, the line should be
libraryDependencies += "org.openlaw" %% "openlaw-core" % "<last version>"

Reply:
Ah of course, fixed it now